gammapy/catalog/core.py

```diff
@@ -274,7 +274,7 @@ def _lookup_extended_source_idx(self):
 
     @property
     def positions(self):
-        """Source positions (`~astropy.coordinates.SkyCoord`)."""
+        """Source positions as a list of `~astropy.coordinates.SkyCoord` objects."""
```

Comment:
```suggestion
        """Source positions a `~astropy.coordinates.SkyCoord` object."""
```
it's not a list of SkyCoord objects but only one that includes a list of coordinates